yellowme/youst-kotlin
# Feature: Lint hook

## What does this PR do?
- Activate **ktlint** linter (https://ktlint.github.io/)
- Reformat Kotlin code throughout the whole project
- Add pre-commit hook to run ktlint

## How should this be manually tested?
Try to commit some changes that have bad formatting (example, more than one consecutive newlines). The pre-commit hook should run and give an error, not letting you commit.
